e2e-tests/portal/cypress/e2e/journeys/maker-bond/bond-form-values.js

```diff
@@ -1,13 +1,26 @@
-const { roundNumber } = require('../../../../../../portal-api/src/utils/number');
-const { oneMonth, today } = require('../../../../../e2e-fixtures/dateConstants');
+import { BOND_FACILITY_TYPE } from '@ukef/dtfs2-common';
+import { oneMonth, today } from '../../../../../e2e-fixtures/dateConstants';
 
-const DETAILS = {
+const roundNumber = (value, digits) => {
+  let modifiedValue = value;
+  let d = digits;
+
+  if (!digits) {
+    d = 2;
+  }
+
+  modifiedValue *= 10 ** d;
+  modifiedValue = Math.round(modifiedValue);
+  modifiedValue /= 10 ** d;
```

Comment:
do we need this? or is there an easier way to rewrite this.  Appreciate it's just been moved but no idea what it is trying to do

Reply:
Can't we use .toFixed()? or Math.round()

Reply:
Agree - see comment here https://github.com/UK-Export-Finance/dtfs2/pull/3851/files#r1829535233

Reply:
I had a play - it's not straight forward to replace or improve this. 

I have added a new ticket to the backlog to improve this: EMS-4002

This PR has been around for a while, don't want to make this any larger.

Reply:
Resolving this since Zain is away. It's since been removed.